rust/src/dcerpc/dcerpc.rs

```diff
@@ -376,6 +378,8 @@ impl DCERPCState {
             tc_ssn_gap: false,
             ts_ssn_trunc: false,
             tc_ssn_trunc: false,
+            req_done: false,
```

Comment:
they are initialized to `false` but can never again be false after they've once been `true`.

In general I don't like this indirect approach. I would much prefer a solution where `AppLayerParserTriggerRawStreamReassembly` is called the moment a req or resp is marked as done. So probably a small method that calls `AppLayerParserTriggerRawStreamReassembly` when `tx::req_done` is set?

Reply:
OK thanks.